src/components/console/views/ModelErrors.svelte

```diff
@@ -0,0 +1,104 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import type { ModelLog, ModelSlim } from '../../../types/model';
+  import { getModelStatusRollup } from '../../../utilities/model';
+  import ModelStatusRollup from '../../model/ModelStatusRollup.svelte';
+  import TabPanel from '../../ui/Tabs/TabPanel.svelte';
+  import { ConsoleContextKey } from '../Console.svelte';
+
+  export let model:
+    | Pick<ModelSlim, 'refresh_activity_type_logs' | 'refresh_model_parameter_logs' | 'refresh_resource_type_logs'>
+    | undefined;
+  export let title: string;
+
+  let selectedLog: 'activity' | 'parameter' | 'resource' | undefined = undefined;
+  let selectedModelLog: ModelLog | null = null;
+
+  $: {
+    const { activityLog, activityLogStatus, parameterLog, parameterLogStatus, resourceLog, resourceLogStatus } =
+      getModelStatusRollup(model);
+
+    if (activityLogStatus === 'error') {
+      selectedLog = 'activity';
+      selectedModelLog = activityLog;
+    } else if (parameterLogStatus === 'error') {
+      selectedLog = 'parameter';
+      selectedModelLog = parameterLog;
+    } else if (resourceLogStatus === 'error') {
+      selectedLog = 'resource';
+      selectedModelLog = resourceLog;
+    }
+  }
+
+  function onSelectCategory(event: CustomEvent<ModelLog | null>) {
+    const { detail: value } = event;
+    selectedModelLog = value;
+  }
+</script>
+
+<TabPanel tabContextKey={ConsoleContextKey}>
+  <div class="model-errors-container">
+    <div class="console-header">
+      <div class="console-title">{title}</div>
+      <div class="model-statuses">
+        <ModelStatusRollup {model} selectable {selectedLog} flow="horizontal" on:select={onSelectCategory} />
+      </div>
+    </div>
+    <div class="errors">
+      {#if selectedModelLog && !selectedModelLog.success}
+        <div class="error">
+          <div class="reason">
+            <div>
+              {selectedModelLog?.error_message}
```

Comment:
If reason is null could we have a default string or hide this?
<img width="739" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/663b035c-0ced-4d6a-a78c-c9d2081bb31e">

Reply:
How were you able to get that error state?

Reply:
Uploaded an old clipper model where only mission model param extraction seemed to fail, can send it to you if that'd be helpful?